### ISSUE TITLE
Icon in dock does not change when different app selected

### ISSUE THREAD

**Issue by mcphail** [sirredbeard/unofficial-webapp-office#13]
Ubuntu Mate 19.10. Snap version 0.2.

Open "Word" from the menu. Click on the burger menu in top left corner of app and select "PowerPoint". Icon in dock remains as "Word" rather than switching to "Powerpoint".

I can't, off the top of my head, think how to get around this one.

**Comment by sirredbeard** [comment]
The launcher is so simple it's not really set up to update the name and icon right now.

**Comment by mcphail** [comment]
Yes. Definitely one for the backburner rather than a high priority fix :)
